array-manipulation-2/src/take-right.js

```diff
@@ -0,0 +1,14 @@
+/* exported takeRight */
+function takeRight(array, count) {
+  var takeArray = [];
+  var giveArray = [];
+  for (var i = 0; i < array.length; i++) {
+    var newValue = (array.length - count);
+    if (i < newValue) {
+      takeArray.push(array[i]);
+    } else {
+      giveArray.push(array[i]);
+    }
```

Comment:
You can simplify here by starting `i` at `array.length - count` and ignoring (removing) your `takeArray`